docs/UserGuide.md

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
Please don't change the file name, other repos also use user_guide.md and it would be good to have them all the same, for ease of working on different projects.

Reply:
see above ;-)